deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -300,12 +284,81 @@ spec:
         name: statefulsets.apps
         version: v1
       version: v1alpha1
-  description: |2-
+  description: |2
+
+    Red Hat Openshift Container Storage (RHOCS) provides hyperconverged storage for applications within an Openshift cluster.
+
+    ## Components
+
+    RHOCS deploys 3 operators.
+
+    ### OCS operator
+
+    The OCS operator is the primary operator for RHOCS. It serves to facilitate the other operators in OCS by performing administrative tasks outside their scope as well as watching and configuring their CustomResources.
+
+    ### Rook
+
+    Rook[1][1] deploys and manages Ceph on Openshift, which provides block and file storage.
```

Comment:
Why are the links here twice?

Reply:
I did it wrong. Should have been `[Rook][1]`.